### ISSUE TITLE
This add-on does not work with non-english chars

### ISSUE THREAD

**Issue by hurvajs** [tijn/gravatars#3]
The add-on does not work with foreign chars. For example it is not able to show correctly chars: "žěščř"... etc. 
An screenshot example is here:

![gravatar add-on bug](https://cloud.githubusercontent.com/assets/14957815/18170338/4a61a6ba-705e-11e6-9266-411b3ca8b545.jpg)

**Comment by tijn** [comment]
Mhh... I see. It's working fine for me though. I have an email from someone in my inbox who's last name is Šimičić. I just verified that this name is correctly right in Thunderbird on Linux and OS X. Are you using Windows?

Do you have any thoughts about a possible reason for this error? I am at a loss for ideas. It's not like I let the add-on rewrite the name or anything, it's just passing on strings that it reads from the Thunderbird API.

**Comment by hurvajs** [comment]
Hi Tijn,
I'm working on Windows (currently tested on Win 10 but I will test it on Win XP as well) with Czech version of TB.
No idea how to debug it, I'm not experienced in TB or JS development. Maybe you can prepare an internal development version with detailed logging and I can test it.

**Comment by tijn** [comment]
Yeah, I see, Windows and character sets... big sigh. I really don't know but I guess the solution should be to set every charset option in every piece of software to UTF-8 if possible. It's weird though since the plugin is not rewriting any string encodings or so, it's just passing on some information from another API as you can see in `getCellText` (gravatars.js line 8).

I would gladly add more logging to the plugin, except that I don't know how to do this. I've been searching for it though since I really missed it during development.

**Comment by bmix** [comment]
Same problem here:

![image](https://user-images.githubusercontent.com/310157/56388281-e7dfe480-6226-11e9-8703-ca6a1324edcc.png)

I am on Windows 10, TB 60.6.1 (32-Bit)

I got no clue about the workings in the background, but since TB is using XML for the GUI, did you check, that the XML has the same encoding (also in the XML prolog's declaration) like the rest of your code?

**Comment by tijn** [comment]
> did you check, that the XML has the same encoding (also in the XML prolog's declaration) like the rest of your code?

I certainly didn't, but did **you**? 😄 I mean, it's all there for everyone to see and it's only a handful of files.

And as always: Send pull requests.
